specs/CustomStoragePartition.md

```diff
@@ -0,0 +1,208 @@
+Custom Storage Partition
+===
+
+# Background
+For certain WebView host apps, there is a desire to have different contexts for
+different WebViews using the same profile so that when the same websites run in
+these WebViews, they will have different DOM storages and cookie jars. An example
+is different accounts within an application.
+Previosly, the application can use different profiles for different contexts, but
+that has 2 short comings:
+- WebViews from different profiles are not allowed to have opener/opened window relationship.
+- Using profiles means totally different storage for all files like http caches and less performance.
+
+To help managing different application contexts while using the same profile, we are
+introducing an API to set custom storage parition and an API to clear all data in
+a custom storage partition.
+
+When an application sets custom storage parition for a WebView, the sites and iframes
+running inside the WebView will act as if the site were running in a third party iframe
+inside a top level site uniquelly associated with the custom storage partition id and have
+separate [storage partition](https://developer.chrome.com/docs/privacy-sandbox/storage-partitioning/)
+and [cookie partition](https://developer.chrome.com/docs/privacy-sandbox/chips/).
+
+# Conceptual pages (How To)
+This feature depends on [storage partition](https://developer.chrome.com/docs/privacy-sandbox/storage-partitioning/)
+and [cookie partition](https://developer.chrome.com/docs/privacy-sandbox/chips/) browser
+features which are currently experimental. Before those features are enabled by default,
```

Comment:
Just curious is "enabled by default" how browser features talk about finalizing experimental behavior?  I don't know if the enabled/disabled Feature-Flags style language is how experimental changes are described in wv2.

Reply:
That's how we discuss it, at least internally. Definitely open to describe it in a way that everyone can understand it easier.

Reply:
No change necessary.